### ISSUE TITLE
Double-click to install encountered an error

### ISSUE THREAD

**Issue by Autumn-one** [samyeyo/LuaRT#194]
The error content is as follows：
---------------------------
Runtime error
---------------------------
__mainLuaRTStartup__.lua:1: module 'install' not found:
	no field package.preload['install']
	no file 'D:\app\lua\lua\install.lua'
	no file 'D:\app\lua\lua\install\init.lua'
	no file 'D:\app\lua\install.lua'
	no file 'D:\app\lua\install\init.lua'
	no file 'D:\app\lua\5.4\share\lua\5.4\install.lua'
	no file 'D:\app\lua\5.4\share\lua\5.4\install\init.lua'
	no file '.\install.lua'
	no file '.\install\init.lua'
	no file 'C:\Users\71887\AppData\Roaming\luarocks\share\lua\5.4\install.lua'
	no file 'C:\Users\71887\AppData\Roaming\luarocks\share\lua\5.4\install\init.lua'
	no file ''
	no file 'D:\app\lua\install.dll'
	no file 'D:\app\lua\5.4\lib\lua\5.4\install.dll'
	no file 'D:\app\lua\loadall.dll'
	no file '.\install.dll'
	no file 'C:\Users\71887\AppData\Roaming\luarocks\lib\lua\5.4\install.dll'
	no file 'D:\app\lua\lua\install.lua'
	no file 'D:\app\lua\lua\install\init.lua'
	no file 'D:\app\lua\install.lua'
	no file 'D:\app\lua\install\init.lua'
	no file 'D:\app\lua\5.4\share\lua\5.4\install.lua'
	no file 'D:\app\lua\5.4\share\lua\5.4\install\init.lua'
	no file '.\install.lua'
	no file '.\install\init.lua'
	no file 'C:\Users\71887\AppData\Roaming\luarocks\share\lua\5.4\install.lua'
	no file 'C:\Users\71887\AppData\Roaming\luarocks\share\lua\5.4\install\init.lua'
	no file ''
---------------------------
确定   
---------------------------

**Comment by samyeyo** [comment]
Thank you for this report.
I think it's an antivirus issue. Maybe try to reinstall with your antivirus disabled ?

**Comment by Autumn-one** [comment]
Thank you for this report. I think it's an antivirus issue. Maybe try to reinstall with your antivirus disabled?

This is not caused by the antivirus program. I checked the path listed in the error message and it does not exist. I don't know if it is related to this

**Comment by samyeyo** [comment]
You have a strange CPATH and PATH environment values, I think it might be related to this.

Try to remove them before installation to see if it works

**Comment by Autumn-one** [comment]
> You have a strange CPATH and PATH environment values, I think it might be related to this. Try to remove them before installation to see if it works
> 
> Btw, I still think your antivirus prevent any file extraction in your current temporary folder...

Thank you, the program is running normally after I removed the LUA_PATH environment variable.

**Comment by Autumn-one** [comment]
done

**Comment by YaroslavSin** [comment]
Not handy solution to remove env variable. ZeroBrainStudio, Lua for Widows created two variables LUA_CPATH, LUA_PATH. Remove one of these variables will break the software.

**Comment by samyeyo** [comment]
I agree. I will check what can be done to bypass LUA_PATH and LUA_CPATH values with executables compiled with `rtc` like the LuaRT installer
